portal/content/docs/contributing/contribution-guidelines.md

```diff
@@ -0,0 +1,76 @@
+
+## Documentation
+
+### Improving documentation
+[Documentation](https://opendataeditor.okfn.org/documentation/getting-started/) is always the best place to start contributing. If you spot a typo or you’ve ever thought, "I wish the documentation explained this more clearly," you can help make it easier to understand. You can either flag that, or propose a modification. You can also add missing steps or explanations. 
+
+### Adding examples
+You can also contribute by adding examples! If you notice something is not covered in the documentation, try creating a helpful example and sharing it. You can also provide real-world use cases to show how Open Data Editor can be used. Every improvement makes Open Data Editor easier for everyone to use!
+
+All documentation files are contained [in this folder](https://github.com/okfn/opendataeditor/tree/main/portal/content/docs/documentation). You can just propose a change or add examples by opening a PR. Please **always include a short description of what you changed and why**, so it is easier to review it. If you don’t feel comfortable using Github, you can also send us an email with the proposed changes at info[at]okfn.org. 
+
+### Case studies
+If you have started using Open Data Editor in your daily work, and would like to share with the world how the tool is helping you, you can always write a blog about it. If possible, try to include photos and screenshots. If you need guidance, you can always reach out to us at info[at]okfn.org. You can send us your blog at that same email address.
+
+## Translations
+You can translate the documentation and/or the app in another language. To do so:
+
+To translate the documentation [...]
+
+To translate the application [...]
+
+## Reporting a bug
+We use Github as a code and issues hosting platform. To report a bug or propose a new feature, please open an issue [in the repository](https://github.com/okfn/opendataeditor/issues). Please provide a detailed description of the bug and include screenshots. There are a few predefined issue templates to help you get started.
+
+If you don’t feel comfortable using Github, you can also send us an email at info[at]okfn.org. 
+
+## Code contributions 
+
+### Coding standards
+If you wish to contribute to the Open Data Editor codebase, the first thing you should do is check [the development contribution guide](https://opendataeditor.okfn.org/contributing/development/) to make sure you have all the prerequisites in place. 
+
+Follow [the Open Knowledge coding standards](https://github.com/okfn/coding-standards) as much as possible, to make sure your code contribution is consistent, readable, and maintainable.
+
+When making changes to an existing file, please try to follow the same style that file already uses. This helps maintain a consistent look and ensures contributions remain focused on meaningful improvements rather than formatting adjustments.
```

Comment:
@sapetti9 this coding standards have more than 6 years and we are not applying them so I would remove this section.